DemoUI/index.html

```diff
@@ -187,12 +194,43 @@
           <i class='fa fa-angle-left backbutton' id='backup-detail-done-button'></i>
         </menu>
         <h1 id='detail-page-title'>EMPTY !!</h1>
-      </header>
+
+</header>
+
+
+      
 
       <section class='gaia-list fit scroll sticky' id='main-page' style='margin-top: 5rem; position: absolute'>
```

Comment:
We should probably change the id for this at some point. I'm pretty sure this is my own inattentive copy/paste at work